src/ui/widgets/InputTraceWidget/InputBars/Bar/Bar.module.scss

```
@@ -38,3 +40,23 @@
 .noRadius {
   border-radius: 0;
 }
+
+.value {
+  position: absolute;
+  top: $space-xs;
+  left: 0;
+  right: 0;
+  text-align: center;
+  font-family: $font-mono;
+  // The bar keeps its width whether the readout is on or not, so three
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)